src/workerd/server/workerd-api.c++

```diff
@@ -398,8 +398,9 @@ kj::Maybe<jsg::ModuleRegistry::ModuleInfo> WorkerdApi::tryCompileModule(jsg::Loc
               lock, Impl::compileJsonGlobal(lock, module.getJson())));
     }
     case config::Worker::Module::ES_MODULE: {
+      // TODO(soon): Make sure passing nullptr to compile cache is desired.
```

Comment:
todo

Reply:
@jasnell any thoughts on this? what's the code-path that triggers this particular case?

Reply:
One such example is workerd-api.c++:557-558 where we compile PYTHON_ENTRYPOINT.
Would've been nice if that was cached.
But it's all relevant in workerd-api only which is only used for local dev so I don't think it matters all too much.

Reply:
This is appropriate here. We are not yet enabling compile cache for anything but built-ins. The TODO could be reworded accordingly but this is fine also